spec/hiptest_publisher_spec.rb

```diff
@@ -17,7 +17,7 @@
 
   after(:each) {
     if @output_dir_created
-      FileUtils.rm_rf(output_dir)
+      #FileUtils.rm_rf(output_dir)
```

Comment:
Was this meant to be commented out? 

(`#` is a comment in this context, right?)